packages/webapp/src/components/collection/cardCollectionDisplay.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import Image from 'next/image'
+import { Card } from 'src/store/types' 
+import { MintDeckModal } from 'src/components/modals/mintDeckModal'
+
+interface CardCollectionDisplayProps {
+  cards: Card[]
+  isHydrated: boolean
+  setSelectedCard: (card: Card | null) => void
+  onCardToggle: (card: Card) => void
+  selectedCards: Card[] 
+  isEditing: boolean
+}
+
+const CardCollectionDisplay: React.FC<CardCollectionDisplayProps> = ({ cards, isHydrated, setSelectedCard, selectedCards, onCardToggle, isEditing }) => {
+  return (
+    <>
+      <div className="col-span-7 flex rounded-xl border overflow-y-auto">
+        {isHydrated && cards.length === 0 && (
+          <div className="flex flex-row w-full justify-center items-center">
+            <MintDeckModal/>
+          </div>

```

Comment:
Whenever clicking some of the filter items in the left panel, this is shown. Clearly not what we want (because the filtered set size is 0).

The condition here should be changed to display this modal only if the full card collection (not the filtered one) is empty.

Btw, I want to refactor the left panel according to this later: https://github.com/0xFableOrg/0xFable/issues/93 and remove these filter buttons. But I think it's good to keep the generic filtering ability. This (refactor left panel) is for a further PR!